doc/development/building.rst

```diff
@@ -0,0 +1,16 @@
+Building Applications
+=====================
+
+`Jenkins <https://tssw-ci.lsst.org/>`_ is the chosen Continuous Integration platform.
+Each application should have a build project in Jenkins.
+These applications should then have a dedicated build server.
```

Comment:
No sure I understand what this phrase means... Are you trying to say we have to select one of the Jenkins nodes to run the CI? I think in general we leave it float to the nodes so Jenkins can schedule them at will, except for the conda recipes, which have a dedicated node.

Reply:
I didn't make any changes to this file, I merely moved text around. So I don't understand the sentence either hehe. However, I modified the sentence to reflect the fact that Jenkins chooses the node to run the build on.